.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Nit: Not a fan of the negative connotation with the example `Ex. I'm always frustrated when` :D

Reply:
For a feature/ and maybe also a bug we could ask if this is a blocker.

Reply:
Will change, some of these blobs are Github defaults